packages/core/src/components/progress/progress.stories.ts

```diff
@@ -0,0 +1,77 @@
+import { htmlMatrix, Meta, omit, Story } from '../../../../../docs';
+import { Progress, ProgressProps } from './progress.story';
+
+const size = ['regular', 'large'] as const;
+const hideLabel = [false, true] as const;
+const children = ['', 'Custom label'] as const;
+
+export default {
+  title: 'Core/Progress',
+  component: Progress,
+  argTypes: {
+    ...omit<ProgressProps>('aria-valuetext'),
+    children: {
+      description: 'Optional label.',
+    },
+    value: {
+      table: {
+        defaultValue: { summary: '0' },
+      },
+    },
+    min: {
+      table: {
+        defaultValue: { summary: '0' },
+      },
+    },
+    max: {
+      table: {
+        defaultValue: { summary: '100' },
+      },
+    },
+    size: {
+      control: { type: 'radio', options: size },
+      table: {
+        type: { summary: size.toString() },
+        defaultValue: { summary: 'text' },
```

Comment:
...and same here, it should be "regular" a default.

Reply:
Done, thanks!